### ISSUE TITLE
Toggle FullScreen key works only during gameplay

### ISSUE THREAD

**Issue by Pulfer** [hrydgard/ppsspp#7542]
1. Define Toggle FullScreen key in settings.
2. Defined Toggle FullScreen key works only during gameplay. 

PPSSPP 1.0.1, SDL2, Linux.

**Comment by hrydgard** [comment]
Hm, yeah, we only run the keymapper ingame... Good point, needs fixing.

**Comment by unknownbrackets** [comment]
Is this the same as #4981?

-[Unknown]

**Comment by Pulfer** [comment]
Seems to be the same.

**Comment by Translator5** [comment]
Also the toggle Fullscreen key-function & the menu-item do not sync!
If you toggle fullscreen using the shortcut-key, menu show you it is still in window mode!
"Linux Mint 18.1"

Thanks for the great application!

**Comment by vnctdj** [comment]
This issue is still present in the latest build as of today.
I would label it accordingly, but I can't... (SDL, Linux)

**Comment by orbea** [comment]
@hrydgard In 2015.

> Hm, yeah, we only run the keymapper ingame... Good point, needs fixing.

I hope this gets some attention eventually...

**Comment by benderscruffy** [comment]
has this been implemented ???

**Comment by hrydgard** [comment]
No, we still don't run the keymapper in the menu. We do have a fixed key though, F11, which does the trick but if you want to toggle fullscreen in the menu from a controller right now, you're still out of luck, as this hasn't been prioritized.
